extensions/amp-accordion/1.0/amp-accordion.css

```diff
@@ -39,11 +39,15 @@ amp-accordion > section > * {
 }
 
 /*
+ * This media query is used instead of @supports due to low compatibility.
+ * At the time of this change (4/6/2021) no version of Webkit support @supports queries.
+ *
  * Note! This is a new CSS incompatible with older browsers. It provides
  * low-specificity UI styles. The same styles are later set for older browsers
  * via the `i-amphtml-accordion-header` class.
  */
-@supports selector(:where(A)) {
+ 
+@media (min-width: 1px) {
   :where(amp-accordion > section) > :first-child {
     cursor: pointer;
```

Comment:
```suggestion
```

Better to wait for `cursor:pointer` until the script adds the `.i-amphtml-accordion-header` class, as that is the point at which the element is actually interactive.

Reply:
Good point! Resolved with update.